pkg/backup/item_backupper.go

```diff
@@ -160,6 +162,29 @@ func (ib *defaultItemBackupper) backupItem(logger logrus.FieldLogger, obj runtim
 	}
 	ib.backedUpItems[key] = struct{}{}
 
+	// We don't want to back up completed jobs or pods
+	if groupResource == podsGroupResource {
+		phase, _, err := unstructured.NestedString(obj.UnstructuredContent(), "status", "phase")
+		if err != nil {
+			return err
+		}
+		if phase == "Failed" || phase == "Succeeded" {
```

Comment:
"k8s.io/api/core/v1".PodFailed/PodSucceeded

Reply:
FYI it's worth checking to see if we're already importing an external package if all we need is a constant or two. You never know how much of an impact (size wise) importing a new package may be.

In this case, the pod command executor already imports this package. If it didn't, I'd recommend creating our own constants instead of importing.